### PR TITLE
Add (default value) as an optional parameter to the Get function.

### PR DESCRIPTION
**Pull request by ayman-alkom**
### Add the ability to set the default value to the get function like the following:
```javascript
localStorageService.get('some_key','def_val');
```
### Old code
```javascript
	var	autoLogin = localStorageService.get('Auth.autoLogin');
    
    if (autoLogin == null) {
    	localStorageService.set('Auth.autoLogin', false);
    	autoLogin = false;
    }
```
### New code
```javascript
	var	autoLogin = localStorageService.get('Auth.autoLogin',false);
```

### CODE REVIEW

**Comment by a8m** [comment]
Can you please create tests for this changes ? 
Thanks @ayman-alkom

File: src/angular-local-storage.js [hunk 1]
```diff
@@ -162,7 +162,12 @@ angularLocalStorage.provider('localStorageService', function() {
       // angular.toJson will convert null to 'null', so a proper conversion is needed
       // FIXME not a perfect solution, since a valid 'null' string can't be stored
       if (!item || item === 'null') {
-        return null;
+				if(defaultValue){
+					addToLocalStorage(key,defaultValue);
+					return defaultValue;
+				}else{
+					return null;
+				}
```
Comment (a8m) [inline src/angular-local-storage.js]: Can you please be consistent with the same code-style we have today ?
thx

**Comment by ayman-alkom** [comment]
I am sorry but this is my fist pull request ;)

**Comment by ayman-alkom** [comment]
I am sorry but I don't know how to create tests
@a8m

File: src/angular-local-storage.js (hunk 1 above)
Comment (ayman-alkom) [inline src/angular-local-storage.js]: I tried to write it again

File: src/angular-local-storage.js [hunk 2]
```diff
@@ -162,7 +162,12 @@ angularLocalStorage.provider('localStorageService', function() {
       // angular.toJson will convert null to 'null', so a proper conversion is needed
       // FIXME not a perfect solution, since a valid 'null' string can't be stored
       if (!item || item === 'null') {
-        return null;
+        if(defaultValue !== null && defaultValue !== undefined){
```
Comment (a8m) [inline src/angular-local-storage.js]: Can you please use `isUndefined` instead ?
(and also add spaces. e.g: `if (expression) { ... }`)

**Pull request closed without merging**